cmd/criticize/main.go

```diff
@@ -144,6 +152,33 @@ func parseArgv(l *linter) {
 	}
 }
 
+func (l *linter) LoadConfig() {
+	if _, err := os.Stat(l.configFile); err != nil {
```

Comment:
Is `os.IsNotExist` check redundant here?

Reply:
Maybe..we should just set default config params as an empty string rather than `.gocritic.json`

Reply:
Probably. Not critical for the first patch.